_includes/ocrtutorial.md

```diff
@@ -0,0 +1,92 @@
+## OCR Tutorial
+
+This tutorial explains the [Optical Character Recognition (OCR) demo](https://github.com/nimbella/demo-projects/tree/master/ocr) available on GitHub and shows you how to deploy it to the Nimbella Cloud.
+
+The OCR application displays a web page where visitors select a language (English, Spanish, or Chinese) and then drag a JPG, GIF, or PNG image onto the page, where it is automatically converted to text. If the conversion is successful, users can read the text and click a **Speak** button to hear the text. You can [try out the application here](https://ocrdemo-apigcp.nimbella.io).
+
+The OCR demo is an example of a serverless Nimbella Cloud application with a [React front end](https://reactjs.org) and several functions on the back end. The [Tesseract OCR Engine](https://github.com/tesseract-ocr/tesseract) converts the images to text.
+
+The OCR demo has the following code and configuration components, described in more detail later:
+
+-Application logic: The _packages_ directory contains several Nimbella actions, which are logical collections of functions.
+- Static web content: The _web_ directory contains a _public_ directory, with the files that will be part of the public website, and a _src_ directory with React component files and other JavaScript files that are built into the _index.html_ template page to produce the final static public page.
+- Slack application: A Nimbella Commander project folder called _commander_, with an _ocr.js_ file with an app that can be installed into Slack.
+
+- A single index.html file, which has a field for a visitor to enter some text and click **Submit**.
+- A single JavaScript file that provides the backend logic for the conversion of text to QR code.
+- A Node package manager file called *package.json*, which describes what dependencies the function has.
```

Comment:
```suggestion
```

Reply:
I removed these 3 bullets as they are from the qr code example.

You should see a "Commit suggestion" button like in this screen shot (greyed out for me because I don't have permission to edit your commits).

<img width="786" alt="Screen Shot 2020-03-09 at 12 33 25 AM" src="https://user-images.githubusercontent.com/4959922/76183102-a426fe00-619d-11ea-997f-1f174a8a76c0.png">